### PR TITLE
docs: update dart frog description

### PR DESCRIPTION
**Pull request by kaiceyd**
## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: docs/docs/overview.md [hunk 1]
```diff
@@ -58,9 +58,10 @@ dart pub global activate dart_frog_cli
 ```
 
 :::info
-[Dart Frog][dart_frog_link] is fast, minimalistic backend framework for Dart.
+[Dart Frog][dart_frog_link] is a stable, fast, minimalistic backend framework for Dart.
```
Comment (scarletteliza) [inline docs/docs/overview.md:61]:
```suggestion
[Dart Frog][dart_frog_link] is a fast, minimalistic backend framework for Dart. It is stable as of [v0.1.0](https://github.com/VeryGoodOpenSource/dart_frog/releases/tag/dart_frog-v0.1.0).
```

File: docs/docs/overview.md [hunk 2]
```diff
@@ -58,9 +58,10 @@ dart pub global activate dart_frog_cli
...
+
+You'll use Dart Frog as a [backend for frontends (BFF)](https://medium.com/mobilepeople/backend-for-frontend-pattern-why-you-need-to-know-it-46f94ce420b0), which allows you to connect your existing backend to the Flutter News Template frontend. Dart Frog reduces the barrier for entry for all publishers, despite backend, and brings your app to market faster without required UI modifications.
```
Comment (scarletteliza) [inline docs/docs/overview.md:63]:
```suggestion
You'll use Dart Frog as a [backend for frontends (BFF)](https://learn.microsoft.com/en-us/azure/architecture/patterns/backends-for-frontends), which allows you to connect your existing backend to the Flutter News Template frontend. Dart Frog reduces the barrier for entry for all publishers, despite backend, and brings your app to market faster without required UI modifications. 
```

**Review by felangel — DISMISSED** [review]
LGTM 🚀

File: docs/docs/overview.md [hunk 3]
```diff
@@ -58,9 +58,10 @@ dart pub global activate dart_frog_cli
 ```
 
 :::info
-[Dart Frog][dart_frog_link] is fast, minimalistic backend framework for Dart.
+[Dart Frog][dart_frog_link] is a fast, minimalistic backend framework for Dart. It is stable as of [v0.1.0](https://github.com/VeryGoodOpenSource/dart_frog/releases/tag/dart_frog-v0.1.0).
+
```
Comment (felangel) [inline docs/docs/overview.md:62]: Looks like we need to format the docs (probably some whitespace issue). Lmk if you need help 👍

**Review by simpson-peter — APPROVED** [review]
lgtm

**Pull request merged**